function_scope_lab.js

```diff
@@ -0,0 +1,304 @@
+const totalNum = (num1, num2) => {
+  return num1 + num2
+}
+let result4 = totalNum(14, 5);
+console.log(result4);
+
+// ## Question 5
+//
+// Write a function takes in any number grade and returns a corresponding
+//letter grade.
+//
+// | Number Grade | Equivalent Letter Grade |
+// | :--------: | :---------: |
+// | 100 | A+ |
+// | 90 - 99 | A |
+// | 80 - 89 | B |
+// | 70 - 79 | C |
+// | 65 - 69 | D |
+// | Below 65 | F |
+//
+//
+// Example:
+//
+// Input: `88`
+//
+// Output: `"B"`
+
+console.log('----Q5----')
+
+function grade(num) {
+  if (num === 100) {
+    return "A+"
+  } else if (num >= 90) {
+    return "A"
+  } else if (num >= 80) {
+    return "B"
+  } else if (num >= 70) {
+    return "C"
+  } else if (num >= 65) {
+    return "D"
+  } else {
+    return "F"
+  }
+  }
+  let result5 = grade(78);
+  console.log(result5);
+
+
+  // ## Question 6
+  //
+  // Implement a function named `repeatLog` that takes a string and a
+  //number as parameters. The function should log `message` `count`
+  //number of times.
+  //
+  // Example:
+  //
+  // Input: `"+", 10`
+  //
+  // Output: `++++++++++`
+
+console.log('----Q6----')
+const repeatLog = (num) => {
+  return str.repeat(num)
+}
+  let str = "+"
+  let result6 = str.repeat(10);
+console.log(result6);
+
+
+  // ## Question 7
+  //
+  // Write a function named `first` that takes a number and returns an array
+  //with all the numbers from 1 to n
+  //
+  //
+  // Example:
+  //
+  // Input: `first(3)`
+  //
+  // Output: `[1, 2, 3]`
+
+console.log("----Q7----")
+let arrNum = []
+  const first = (num) => {
+    for (let i = 1; i <= num; i++) {
+      arrNum.push(i)
+      }
+      return arrNum
+      }
+
+    let result7 = first(8);
+    console.log(result7);
+
+// ## Question 8
+//
+// Write a function that logs the numbers from 1 to x, except:
+//
+// If the number if a multiple of 3, log `"Fizz"` instead of the number
+// If the number is a multiple of 5, log `"Buzz"` instead of the number
+// If the number is a multiple of 3 AND 5, log `"FizzBuzz"` instead of the number
+// Your function should take in one parameter: x (the number to count up to)
+
+console.log("----Q8----")
+
+  const lengthNum = (x) => {
+    for (let i = 1; i <= x; i++) {
+    if(i % 3 === 0 && i % 5 === 0) {
+    console.log("FizzBuzz")
+    }
+    else if (i % 5 === 0) {
+      console.log("Buzz")
+    }
+    else if (i % 3 === 0) {
+      console.log("Fizz")
+    } else {
+      console.log(i)
+    }
+    }
+    }
+let result8=lengthNum(20);
+console.log(result8)
+
+  // ## Question 9
+  //
+  // Write a function named `reverse` that takes an array as a parameter.
+  //The function should return an array in reverse order.
+  //
+  //
+  // Example:
+  //
+  // Input: `reverse([1, 2, 3])`
+  //
+  // Output: `[3, 2, 1]`
+
+  console.log('----Q9----')
+  const reverse =(arr) => {
+     return arr.reverse()
+   }
+   let arrInput = [1, 2, 3]
+
+   let result9 = reverse(arrInput);
+   console.log(result9);
+
```

Comment:
Great use of the reverse method! Is their a way to create your own reverse functionality?